lang/cs/Org.Apache.REEF.IO/FileSystem/IFileSystem.cs

```diff
@@ -106,9 +106,10 @@ public interface IFileSystem
         IEnumerable<Uri> GetChildren(Uri directoryUri);
 
         /// <summary>
-        /// Returns Url prefix 
+        /// Create Uri from a given file name
```

Comment:
Please document:

  * What is assumed for `path`? It is relative, right?
  * What are error casses? E.g.:
    * What if `path` is a Uri for a different file system?
    * What if the `path` can't be converted to a valid Uri? (e.g. because of illegal strings)
    * What if `path` is `null` or empty?